server/services/WeekPhsaseService/weekPhasesFunctions.js

```diff
@@ -0,0 +1,36 @@
+const weekPhases = require("./weekPhaseInterval");
+const dayjs = require("dayjs");
+const { WeekPhaseDate } = require("./weekPhaseDate");
+
+exports.getTrueActivePhase = ()=>{
+    let nowjs = dayjs();
+    let now = new WeekPhaseDate(nowjs.day(), nowjs.hour(), nowjs.minute());
```

Comment:
You should hide this in the method. This is not a boundary so this is a logic issue. Build the string as in the previous comments.
If it makes you feel better you can put the building in a function, but it would be pretty much useless.